participants/1pan.md

```diff
@@ -0,0 +1,69 @@
+# <Project Name>
+
+Before submitting a PR containing your team's project information:
+
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+Name:1pan
+Slack:longkengkeng
+
+MemberId:U01BZR3FNPJ
+
+## (1) Brief Project Description
+
+*We wanted to create a disk application on the Filecoin network so that our user data would not be lost. For the foreseeable future, we hope that our application data in the Filecoin gateway will be easily accessible to our users.*
+
+## (2) Link to Project UI
+
+*demo: http://ipfs.forke.cn/*
+
+You can see UI in github:
+
+https://github.com/zlongxiao/1pan.git
+
+## (3) What does your application/UI do?
+
+*This is a traditional disk project, and we really want to see it become a business application on the Filecoin network.The project has login, registration, upload, share, download, reward and other functions.*
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+*Various*
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+*We will not collect any data, and we will slice the data and store it in the Filecoin network. In the IPFS network, there is no complete data in our application, and the data uploaded to the IPFS network will be reviewed by our staff.*
```

Comment:
Hi there, can you please say a little bit more about the data you're planning to store during this competition? From reading your application, it's not clear to me which datasets you will be using and where it is coming from.

Reply:
User's data upload to my application,we will deal in my server to cut files into pieces,and then we send pieces to Filecoin network

Reply:
Our application is a network disk application, we accept any possible data, such as ZIP, MP4, etc